guides/source/getting_started.md

```diff
@@ -242,220 +241,578 @@ different _actions_. Each action's purpose is to collect information to provide
-welcome controller's index action. This was created earlier when you ran the
-controller generator (`bin/rails generate controller Welcome index`).
+This `root` method defines a _root path_ for our application. The `root` method tells Rails to map requests to the root of the application to the `ArticlesController` `index` action.
 
 Launch the web server again if you stopped it to generate the controller (`rails
 server`) and navigate to <http://localhost:3000> in your browser. You'll see the
-"Hello, Rails!" message you put into `app/views/welcome/index.html.erb`,
-indicating that this new route is indeed going to `WelcomeController`'s `index`
+"Hello, Rails!" message you put into `app/views/articles/index.html.erb`,
+indicating that this new route is indeed going to `ArticleController`'s `index`
 action and is rendering the view correctly.
 
 TIP: For more information about routing, refer to [Rails Routing from the Outside In](routing.html).
 
-Getting Up and Running
-----------------------
+## Adding Articles
+
+So far, we have seen routes, controllers, actions and views within our Rails application. All of these are conventional parts of Rails applications and it is done this way to follow the MVC pattern. The MVC pattern is an application design pattern which makes it easy to separate the different responsibilities of applications into easy to reason about pieces.
 
-Now that you've seen how to create a controller, an action, and a view, let's
-create something with a bit more substance.
+So with "MVC", you might guess that the "V" stands for "View" and the "C" stands for controller, but you might have trouble guessing what the "M" stands for. This next section is all about that "M" part, the _model_.
 
-In the Blog application, you will now create a new _resource_. A resource is the
-term used for a collection of similar objects, such as articles, people, or
-animals.
-You can create, read, update, and destroy items for a resource and these
-operations are referred to as _CRUD_ operations.
+### Generating an Article model
 
-Rails provides a `resources` method which can be used to declare a standard REST
-resource. You need to add the _article resource_ to the
-`config/routes.rb` so the file will look as follows:
+A model is a class that is used to represent data in our application. In a plain-Ruby application, you might have a class defined like this:
 
 ```ruby
-Rails.application.routes.draw do
-  get 'welcome/index'
+class Article
+  attr_reader :title, :body
 
-  resources :articles
+  def initialize(title:, body:)
+    @title = title
+    @body = body
+  end
+end
+```
+
+Models in a Rails application are used in the same way: they represent data.
+
+Models have another purpose in a Rails application as well. They're also used to interact with the application's database. Models use a part of Rails called Active Record to interact with the database of the application.
```

Comment:
"Models in a Rails application are used in the same way" and "Models have another purpose in a Rails application as well" is contradicting itself.